android/app/build.gradle

```diff
@@ -87,6 +87,7 @@ dependencies {
     compile 'com.jakewharton.timber:timber:4.1.0'
     compile 'com.jakewharton:butterknife:8.4.0'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
+    compile 'jp.wasabeef:recyclerview-animators:2.2.5'
```

Comment:
Why not use the native animations provided? 
Also I don't think that there is a need for animations anyways.